### ISSUE TITLE
formatting readability

### ISSUE THREAD

**Issue by sparrell** [oasis-tcs/openc2-oc2ls#183]
This is a child of #143

```
Page 40, Line 27: the run-on formatting of the example command message makes it hard to read and reduces its utility as an example.
```

**Comment by davaya** [comment]
The markdown source is formatted properly, but the example needs to be enclosed in triple backticks as with the other examples.

**Comment by romanojd** [comment]
I've addressed this issue within issue #143. Closing this comment.
